### PR TITLE
Base Rules

### PR DESCRIPTION
**Pull request by seanKenkeremath**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,121 @@
...
+* 1 stump
+* 1 hammer
+* 1 nail for each player. These nails must all be the same type
+* 3 nails to be placed in the center of the stump. These nails do not necessarily have to be the same
+* Any number of players
+
+## Setting up
+### Nails
+Begin by hammering 1 nail for each player into the stump. All nails should be placed along the same ring of the stump. 
+All nails should be hammered to an equal, agreed upon depth. Hammer the 3 center nails into the center of the stump.
```
Comment (austenlux) [inline README.md:15]: Maybe clarify here that the 3 center nails are only hammered in the same agreed upon depth?

File: README.md [hunk 2]
```diff
@@ -1 +1,121 @@
...
+
+
+A nail is considered completely hammered into the stump if all plays agree or if the Hammer Test decides it is (see Hammer Test). Any player may challenge a nail and perform a Hammer Test. This effectively pauses the game. Giving away the hammer for someone to perform the Hammer Test does not end your turn, and that player must return the hammer to you upon completing the Hammer Test, resuming the game.
+
+
+Each successful strike on a nail grants the player one drink to give out during the Drink phase. 
+Each miss is one drink the player must take themselves during the Drink phase.
+
+
+If a player hits the wrong nail it counts as a miss. If doing so results in the final center nail being hammered into the stump, A new center nail is created and the current player is removed from the game.
```
Comment (austenlux) [inline README.md:79]: should be lowercase "A" in "A new center nail is created". Also maybe say "a new center nail is hammered back in place" or something? I dunno, just "created" sounds a bit odd to me.

File: README.md [hunk 3]
```diff
@@ -1 +1,121 @@
...
+A Hammer Test can be performed by any player to determine if a nail has been completely hammered into the stump. For a nail that has been hit in vertically, the hammer is turned sideways the head is run over the nail from all angles. The nail is considered completely hammered in if the hammer does not clink with the nail. For a nail hit in sideways the same rule applies except the hammer is run over the nail both ways lengthwise. If everyone agrees a nail is completely hammered in then the Hammer Test can be skipped. Any player may challenge another player's nail during their turn via the Hammer Test.
+
+
+If the Hammer Test is performed during another players turn and that player gives up the hammer, it does not count as forfeiting the hammer; The hammer must be returned to that player after the test is complete.
+
+
+### Demotions
+
+
+If a Lord drops their hammer during the Flip phase they are instantly demoted to Lad. Before ending their turn they must drive in a new starting nail to the center position.
```
Comment (Kritarie) [inline README.md:117]: Shouldn't this be "they must drive in a new starting nail to the outer ring"? Maybe just odd wording, when I see center position I think of the 3 center nails

File: README.md [hunk 4]
```diff
@@ -1 +1,121 @@
...
+
+If a player hits the wrong nail it counts as a miss. If doing so results in the final center nail being hammered into the stump, A new center nail is created and the current player is removed from the game.
+
+
+#### Drink
+A "Drink" is a consistent unit which must be decided upon before the game begins.
+
+
+During the drink phase the player may give out drinks for each successful strike they made during the Strike phase. 
+For each strike made as a Lad, that player may give a drink out to another Lad. For each strike made as a Lord, that player may give a drink out to another Lord or make all remaining Lads drink at once.
```
Comment (austenlux) [inline README.md:87]: Maybe explicitly say here that a Lad can't ever make a Lord drink. It's implied, but might be good to be extra clear.

File: README.md [hunk 5]
```diff
@@ -1 +1,121 @@
...
+The final strike of a starting nail is considered a strike made as a Lord, as that player becomes a Lord simultaneously to striking the nail.
+
+
+The final strike of the game (the last center nail) is considered a strike made as a rank even higher than Lord. In this case all other players must drink.
+
+
+The player then must drink for each missed strike he or she performed during the Strike phase.
+
+
+Drinks may only be given out as long as the current player holds the hammer. If the player gives away the hammer without giving out drinks, their turn ends immediately and they forfeit giving out those drinks. However, they still must drink for each missed strike they made. Giving away the hammer for a hammer test does not count as forfeiting the hammer, and the player performing the hammer test must return the hammer when they are finished.
```
Comment (austenlux) [inline README.md:99]: Maybe capitalize Hammer Test here, since it has been capitalized in other places so far.

**Review by austenlux — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (mcanidae) [reply README.md:117]: Yeah I agree center position definitely makes me think of the three in the middle.

**Pull request closed without merging**